src/app/models/LikeModel.ts

```diff
@@ -0,0 +1,23 @@
+import * as mongodb from 'mongodb';
+
+class LikeModel {
+ _id: mongodb.ObjectId;
+ email: string;
+ planId: mongodb.ObjectId; 
+ createdAt: Date;
+
+ constructor(
+    email = '',
+    planId = '', 
+    createdAt = new Date(),
```

Comment:
we don't allow user to enter the createdAt date themselves so it should not be passed in as a param.

Reply:
Can you check latest version? original included as I was modeling after ClassModel which has createdAt generated/assigned as a param.